CVO_co@30_Hearts_and_Minds.NorthTakistan/initPlayerLocal.sqf

```diff
@@ -1,4 +1,4 @@
-#include "cvo\init\cvo_cbrn_local.sqf"
+#include "cvo\cbrn\initPlayerLocal.sqf"
```

Comment:
is there a reason the file is called initPLayerLocal?
Since its a name thats already is in use in the root, maybe renamingthe file to cbrn_initPlayerLocal.sqf will help avoiding a mix-up

Reply:
Renamed the file to cbrn_init_local.sqf